quinn-h3/src/qpack/decoder.rs

```diff
@@ -447,6 +448,7 @@ mod tests {
     fn build_table(n_field: usize) -> DynamicTable {
```

Comment:
I was a little bit confused by the existence of two functions named `build_table()`, one which takes an argument (this one) and another one (in `dynamic.rs`) which does not. Maybe rename this one to `build_table_with_size()`?